library/src/main/java/com/github/lukaspili/reactivebilling/observable/GetPurchasesObservable.java

```diff
@@ -33,7 +36,10 @@ protected void onBillingServiceReady(BillingService billingService, Observer<? s
         try {
             observer.onNext(billingService.getPurchases(purchaseType, continuationToken));
             observer.onCompleted();
-        } catch (RemoteException e) {
+        } catch (RemoteException | JSONException e) {
+            if(e instanceof JSONException) {
```

Comment:
True, however, when we have a JSONException we want to know that the JSON is invalid